parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -71,6 +71,22 @@ downloadCNI() {
     retrycmd_get_tarball 120 5 "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ${CNI_PLUGINS_URL} || exit $ERR_CNI_DOWNLOAD_TIMEOUT
 }
 
+downloadSecureTLSBootstrapKubeletExecPlugin() {
+    local plugin_download_path="/opt/azure/containers/tls-bootstrap-client"
+    local plugin_version="v0.1.0-alpha.2"
+    local plugin_url="https://k8sreleases.blob.core.windows.net/aks-tls-bootstrap-client/${plugin_version}/linux/amd64/tls-bootstrap-client"
+    if [[ $(isARM64) == 1 ]]; then
+        plugin_url="https://k8sreleases.blob.core.windows.net/aks-tls-bootstrap-client/${plugin_version}/linux/arm64/tls-bootstrap-client"
+    fi
+
+    mkdir -p /opt/azure/containers
```

Comment:
should this not be mkdir -p $plugin_download_path?

Reply:
wait my bad, thats just a binary right

Reply:
no you're right, I actually think it should be

mkdir -p $plugin_download_path as you said, with:

local plugin_download_path="/opt/azure/containers"

Reply:
yeah it's just a binary

Reply:
we could just get the basepath of plugin_download_path and do it that way I guess

Reply:
refactored